src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 13.3.0
```

Comment:
This is partially duplicating and should be merged with the 13.2.0 section. The one addition is "Implement data-driven styling support for `*-pattern properties`".